qiskit/circuit/quantumcircuit.py

```diff
@@ -2265,17 +2265,77 @@ def reset(self, qubit: QubitSpecifier) -> InstructionSet:
         return self.append(Reset(), [qubit], [])
 
     def measure(self, qubit: QubitSpecifier, cbit: ClbitSpecifier) -> InstructionSet:
-        """Measure quantum bit into classical bit (tuples).
+        r"""Measure in Z basis a quantum bit (``qubit``) into a classical bit (``cbit``).
+
+        When quantum state is measured a qubit is project in the computational (Pauli Z) basis to
+        either :math:`\lvert 0 \rangle` or :math:`\lvert 1 \rangle`. The classical bit ``cbit``
+        indicates the result
+        of that projection as a ``0`` or a ``1`` respectively. This operation is non-reversible.
```

Comment:
```suggestion
        r"""Measure a quantum bit (``qubit``) in the Z basis into a classical bit (``cbit``).

        When a quantum state is measured, a qubit is projected in the computational (Pauli Z) basis
        to either :math:`\lvert 0 \rangle` or :math:`\lvert 1 \rangle`. The classical bit ``cbit``
        indicates the result
        of that projection as a ``0`` or a ``1`` respectively. This operation is non-reversible.
```

Reply:
(for the first sentence, what you had is probably technically grammatically correct, it just reads a bit oddly)

Reply:
I'll just commit this in order to move the PR to merge.